generator/internal/rust/templates/crate/src/builder.rs.mustache

```diff
@@ -176,6 +176,10 @@ pub mod {{Codec.ModuleName}} {
         {{#InputType.Codec.SingularFields}}
 
         /// Sets the value of [{{Codec.FieldName}}][{{Codec.FQMessageName}}::{{Codec.SetterName}}].
+        {{#DocumentAsRequired}}
+        ///
+        /// This is documented as a **required** field for requests.
```

Comment:
s/documented as //

too meta for me.

Reply:
here and below.

Reply:
Technically the field may not be required anymore.  The service may have dropped this requirement. You are right though, it is too "Ackchyually".

Fixed.

Reply:
> Technically the field may not be required anymore. The service may have dropped this requirement.

Doesn't matter. The documentation is wrong whether we say "This is ..." vs. "This is documented as ...".

My point was more that these comments are the documentation. I think we were distinguishing between service documentation vs. client library documentation, but customers will not recognize that distinction.